chatterbot/adapters/logic/closest_meaning.py

```diff
@@ -0,0 +1,94 @@
+from .logic import LogicAdapter
+from collections import Counter
+import nltk
```

Comment:
In your tests you should be installing `nltk`, and you probably want to guard against the `ImportError` here and raise a useful message.

No need to require it in your `setup.py`, as it's only needed for this adapter.